src/fswAlgorithms/effectorInterfaces/thrusterPlatformReference/thrusterPlatformReference.h

```diff
@@ -0,0 +1,81 @@
+/*
+ ISC License
+
+ Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+
+#ifndef _THRUSTER_PLATFORM_REFERENCE_
+#define _THRUSTER_PLATFORM_REFERENCE_
+
+#include <stdint.h>
+#include "architecture/utilities/bskLogging.h"
+#include "cMsgCInterface/VehicleConfigMsg_C.h"
+#include "cMsgCInterface/RWArrayConfigMsg_C.h"
+#include "cMsgCInterface/RWSpeedMsg_C.h"
+#include "cMsgCInterface/CmdTorqueBodyMsg_C.h"
+#include "cMsgCInterface/HingedRigidBodyMsg_C.h"
+#include "cMsgCInterface/BodyHeadingMsg_C.h"
+
+#define PLATFORM_REFERENCE_EPS 1e-15
```

Comment:
:small_red_triangle_down: Let's move this into the .c file and declare it as `const double epsilon = 1e-12;` Let's work to move aware from these usages of `#define`

Reply:
Pat, I agree that we are getting too many `#define` statements for epsilon.  With your suggestion they become a public module variable?  Or are you talking about having each method redefine `epsilon`?   I don't like either of these approaches.  As a public variable in a C-module the use must set it to a non-zero value.  If we redefine this in each method then it becomes tedious to maintain. 

Rather, why don't we define `epsilon` in `src/architecture/utilities/macroDefinitions.h`?  Then all the modules needing a generic "small" variable can use this?  That file is for commonly used macro's?

Reply:
So what should I do ultimately? Define it in the .c file?
Also, should I go back and change the EPS variables used in the other modules that have already been merged?

Reply:
Define it in the .c. There's no imperative to go back and change other modules. You're of course welcome to do so in another PR if you want to do some code base spring cleaning :)

Reply:
Done!